### ISSUE TITLE
Send Image

### ISSUE THREAD

**Issue by giovani-junior-dev** [ortod0x/google-messages-api#1]
I would like to know if it is possible to send an image, as I want to help with the project, I have already transformed it into an api rest. I will be publishing it as a way of helping, but first I would like to know if there is a way to send this type of file, if not, you can add it, by showing me the way I can increase it

**Comment by ortod0x** [comment]
hi, sorry for late reply, sure it's possible to send an image, i'll update it later!

**Comment by giovani-junior-dev** [comment]
Thank you very much for the master answer, I'm trying to do it here, but still without success, another thing I'm trying to do is a webhook to receive the messages we receive, if you can make progress or if you can explain to me how you could do it, I'll try to help with your project. But thank you in advance for your contact.

**Comment by ortod0x** [comment]
Hi, maybe you can read https://github.com/ortod0x/google-messages-client/ my lib first before continue your work, i'm using Google Messages Client before i create this API

**Comment by giovani-junior-dev** [comment]
Yes, I am using your lib to build the api. I'm trying to create the webhook scheme using your lib, I've already made it multisession, I'm trying to send images and webhook. If you could give me a tip on how I can move forward with the issue of receiving the SMS we receive.
